app/src/services/permit.ts

```diff
@@ -89,8 +91,21 @@ const service = {
    * Get all Permit types
    * @returns {Promise<PermitType[]>} The result of running the findMany operation
    */
-  getPermitTypes: async () => {
+  getPermitTypes: async (initiative: Initiative) => {
+    const initiativeResult = await prisma.initiative.findFirstOrThrow({
+      where: {
```

Comment:
I think we could make one query here if we want to reduce DB calls, we could include `permit_type_initiative_xref` and then map over `initiativeResult.permit_type_initiative_xref` for the fromPrismaModel.